ground/gcs/src/libs/glc_lib/glc_lib.pro

```diff
@@ -504,7 +504,7 @@ INSTALLS += include_lib3ds include_glext include_quazip include_glc_maths includ
 INSTALLS += include_glc_scengraph include_glc_geometry include_glc_shading include_glc_viewport
 INSTALLS += include_glc_3dwidget include_glc_glu
 
-INSTALLS += target
+!contains(INSTALLS,target):INSTALLS += target

```

Comment:
Done. I've basically just made sure I preserve the current behaviour with each of these fixes, even if it's a bit odd.